rocks-bin/src/project/mod.rs

```diff
@@ -1 +1,315 @@
-pub mod write_new;
+use std::{error::Error, path::PathBuf, str::FromStr};
+
+use clap::Args;
+use eyre::{eyre, Result};
+use inquire::{
+    ui::{RenderConfig, Styled},
+    validator::Validation,
+    Confirm, Select, Text,
+};
+use itertools::Itertools;
+use spdx::LicenseId;
+use spinners::{Spinner, Spinners};
+
+use crate::utils::github_metadata::{self, RepoMetadata};
+use rocks_lib::{package::PackageReq, project::Project};
+
+// General notes and ideas:
+// - Create a subdirectory for the project, unless a path is explicitly passed.
+// - Automatically detect build type by inspecting the current repo (is there a Cargo.toml? is
+//   there something that tells us it's a lua project?).
```

Comment:
question: I suppose these notes are outdated?

Reply:
Yep, the first one is for sure.

Reply:
resolved :)